ttm/releaser.py

```diff
@@ -229,6 +229,17 @@ def is_snapshotable(self):
                                            self.project.product_repo, arch):
                         return False
 
+        # The FTP tree isn't released with setrelease, so it needs to contain
+        # the product version already.
+        product_version = self.get_product_version()
+        if product_version is not None:
+            for product in self.project.ftp_products:
+               for binary in self.binaries_of_product(self.project.name, product):
+                   # The NonOSS tree doesn't include the version...
+                   if binary.endswith('.report') and 'NonOss' not in binary and product_version not in binary:
+                       self.logger.debug(f'{binary} in {product} does not include {product_version}')
```

Comment:
"does not include" or "does not match" ?

Rest seems all right

Reply:
Really "does not include". For Leap, the `ProductVersion` is `15.4`, so fully matching that is not trivial without running into more edge cases.

Reply:
for Leap, OSRT:ProductVersion is hard-set to 15.4; so the check is close to NOP for Leap, but should at least not break